test/RibbonThetaVault.js

```diff
@@ -2665,13 +2691,115 @@ function behavesLikeRibbonOptionsVault(params) {
           depositAmount
         );
 
-        // Only 1 ether - MINIMUM_SUPPLY works
+        // only 1 ether - MINIMUM_SUPPLY works
         await expect(
           this.vault.withdraw(depositAmount.sub(BigNumber.from("1")))
         ).to.be.revertedWith(/Insufficient share supply/);
       });
     });
 
+    describe("#withdrawToV1Vault", () => {
+      time.revertToSnapshotAfterEach();
+
+      it("should fail if vaults aren't registered", async function () {
+        const depositAmount = BigNumber.from("100000000000");
+        await depositIntoVault(
+          params.collateralAsset,
+          this.vault,
+          depositAmount
+        );
+
+        await expect(
+          this.vault.withdrawToV1Vault(
+            this.vault.maxWithdrawableShares(),
+            this.spareVault.address
+          )
+        ).to.be.revertedWith("!feeless");
```

Comment:
The error message `feeless` doesn't really clarify anything, should change it to "Cannot withdraw to vault" or something